integration-tests/tests/circuits.rs

```diff
@@ -5,7 +5,14 @@ macro_rules! declare_tests {
             async fn [<serial_test_evm_ $name>]() {
                 log_init();
                 let block_num = GEN_DATA.blocks.get($block_tag).unwrap();
-                let pk = None;
+                let pk = if $real_prover { Some((*EVM_CIRCUIT_KEY).clone()) } else { None };
+                let (builder, _) = gen_inputs(*block_num).await;
+                let mut block = block_convert::<Fr>(&builder.block, &builder.code_db).unwrap();
+                block.circuits_params.max_evm_rows = 0;
+                let rows_required = EvmCircuit::get_min_num_rows_required(&block);
+                if rows_required > rows_required {
+                    panic!("EVM circuit not enough rows {}, required {}", rows_required, rows_required);
```

Comment:
Are you comparing the same variable here? I guess you wanted to do:
```suggestion
                // This only works if it's not `0`.
                if rows_required > block.circuits_params.max_evm_rows {
                    panic!("EVM circuit not enough rows {}, required {}", rows_required, rows_required);
```

In case we can have `0` here we need to find alternative ways to fetch the runtime computation of rows.

Reply:
So actually the idea is that it should be the constant MAX_EVM_ROWS that is used in the integration tests, fixed